tests/test_accuracy.py

```diff
@@ -56,11 +52,23 @@ def heat(n, deg, time_stages, stage_type="deriv", splitting=IA):
     return errornorm(uexact, u) / norm(uexact)
 
 
-@pytest.mark.parametrize("stage_type", ("deriv", "value"))
+@pytest.mark.parametrize("kwargs", ({"stage_type": "deriv"},
+                                    {"stage_type": "value"}))
 @pytest.mark.parametrize(('deg', 'convrate', 'time_stages'),
                          [(1, 1.78, i) for i in (1, 2)]
                          + [(2, 2.8, i) for i in (2, 3)])
-def test_heat_eq(deg, convrate, stage_type, time_stages):
-    diff = np.array([heat(i, deg, time_stages, stage_type) for i in range(3, 6)])
+def test_heat_eq(deg, convrate, time_stages, kwargs):
+    diff = np.array([heat(i, deg, time_stages, **kwargs) for i in range(3, 6)])
+    conv = np.log2(diff[:-1] / diff[1:])
+    assert (conv > convrate).all()
+
+
+@pytest.mark.parametrize(('deg', 'convrate', 'time_stages'),
+                         [(2, 2.8, i) for i in (2, 3)]
+                         + [(3, 3.8, i) for i in (3, 4)])
+def test_heat_bern(deg, convrate, time_stages):
+    diff = np.array([heat(i, deg, time_stages, **{"stage_type": "value", "basis_type": "Bernstein"}) for i in range(3, 8)])
+    print(diff)
```

Comment:
These `print` statements should be removed for tests

Reply:
Done